src/network/mod.rs

```diff
@@ -0,0 +1,108 @@
+pub mod client;
+pub mod replication;
+pub mod server;
+
+use std::str::FromStr;
+
+use bevy::prelude::*;
+use bevy_egui::{
+    egui::{self},
```

Comment:
I'm surprised this isn't a `clippy` or `rust-analyzer` warning, but this `egui::{self}` can be replaced with just `egui`. Perhaps we should enable import formatting in rustfmt?

Reply:
We can enable this.